rule/error_strings_test.go

```diff
@@ -34,7 +34,8 @@ func TestErrorStringsRule_Configure(t *testing.T) {
 		{
 			name:      "Invalid function",
 			arguments: lint.Arguments{"errors."},
-			wantErr:   errors.New("found invalid custom function: errors."),
+			//nolint:revive
```

Comment:
I think we should adding an explanation we used `//nolint:revive` here.

Reply:
I agree.

We should enforce using an explanation with nolint 

https://github.com/gofr-dev/gofr/blob/dac6bdc735c3b60ccfb17d8e4a799c23e3d7383d/.golangci.yml#L97-L98